coffee_core/src/coffee.rs

```diff
@@ -247,6 +247,7 @@ impl PluginManager for CoffeeManager {
         plugin: &str,
         verbose: bool,
         try_dynamic: bool,
+        _branch: Option<String>,
```

Comment:
```suggestion
        branch: Option<String>,
```

You should pass this to the install command